plugin-dotnet-agent/src/test/kotlin/jetbrains/buildServer/dotnet/test/custom/CustomCommandWorkflowComposerTest.kt

```diff
@@ -73,6 +75,7 @@ class CustomCommandWorkflowComposerTest {
         every { pathObserver.onNext(any()) } returns Unit
         every { _argumentsService.split("args") } returns sequenceOf("arg1", "arg2")
         every { _targetService.targets } returns emptySequence()
+        every { _loggerService.writeBuildProblem(any(), any(), any()) } just Runs
```

Comment:
Could you please also consider to verify this method call through `verify(exactrly = 1) { .... }` construct in this test? Why I'm asking -- this is not just logging but behavioral thing that impacts to user recognisable results in UI